subsys/bluetooth/controller/Kconfig

```diff
@@ -143,6 +143,13 @@ config BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT
 	  microseconds. The event length and the connection interval are the
 	  primary parameters for setting the throughput of a connection.
 
+config BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT
+	int "Default central ACL event spacing [us]"
+	default BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT
+	help
+	  On the central, sets the time ACL connections are spaced apart assuming that
```

Comment:
It would be good to mention when is may be beneficial to set BT_CTLR_SDC_CENTRAL_ACL_EVENT_SPACING_DEFAULT != BT_CTLR_SDC_MAX_CONN_EVENT_LEN_DEFAULT